nestkernel/conn_builder.cpp

```diff
@@ -1376,8 +1376,12 @@ nest::FixedTotalNumberBuilder::connect_()
   librandom::BinomialRandomDev bino( grng, 0, 0 );
 #endif
 
-  for ( int k = 0; k < M; k++ )
+  for ( int k = 0; k < M - 1; k++ )
```

Comment:
@stinebuu You may want to add a comment here to explain why we only need to go to M-1. Otherwise, we will probably wonder about this a year from now.